app/enterprise/2.2.x/developer-portal/administration/application-registration/auth-provider-strategy.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Authorization Provider Strategy for Application Registration
+---
+
+## Overview
+
+In the 1.5.x beta version of the Application
+Registration plugin, the feature was tightly coupled with OAuth2. Kong was the
+only available system of record (SoR) for application credentials and the OAuth
+configuration was done directly within the Application Registration plugin.
+
+In the {{site.ee_product_name}} 2.1.x version, authentication was decoupled from the
+Application Registration plugin. Support has been added for third-party OAuth2
+providers. Developers have the flexibility to choose from either
+Kong or a third-party identity provider (IdP) as the system of record for
+application credentials. With third-party (external) OAuth2 support, developers
+can centralize application credential management with the
+[supported identity provider](/enterprise/{{page.kong_version}}/developer-portal/administration/application-registration/3rd-party-oauth#idps)
+of their choice.
+
+In the {{site.ee_product_name}} 2.2.1.x version and later, support has been added
+for the Key Authentication plugin to use with Kong as the system of record.
+
+### Supported Authentication Plugins
+
+OAuth2 plugins for use with the Application Registration plugin:
+
+- When Kong is the system of record, the Application Registration plugin works
+  in conjunction with the Kong OAuth2 or the Kong Key Authentication plugin.
+
+  <div class="alert alert-warning">
+    <strong>Important:</strong> The Kong OAuth2 plugin does not support
+    hybrid mode. Since the Key Authentication plugin is using the `kong-oauth2` strategy
+    and client ID credential, hybrid mode is also not supported for the Key
+    Authentication plugin. If your organization uses hybrid mode, you must use an external identity
+    provider and configure the Kong OIDC plugin.
```

Comment:
```suggestion
    provider and configure the OpenID Connect plugin.
```
I've noticed that we don't refer to this plugin as OIDC in the docs very often. Might be difficult to find if you use the short form only. If you'd like to keep using the term OIDC, then this should say "configure the OpenID Connect (OIDC) plugin".